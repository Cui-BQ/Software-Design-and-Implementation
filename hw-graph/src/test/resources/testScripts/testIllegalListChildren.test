# Test ListChildren by using Illegal parentNode. e.g. parentNode = null or not exist

# Create graphs
CreateGraph graph

# null lists children
ListChildren graph null

# new lists children
ListChildren graph new