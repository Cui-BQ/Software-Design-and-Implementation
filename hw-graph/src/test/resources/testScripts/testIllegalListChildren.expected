# Test ListChildren by using Illegal parentNode. e.g. parentNode = null or not exist

# Create graphs
created graph graph

# null lists children
Exception: java.lang.IllegalArgumentException: node is null

# new lists children
Exception: java.lang.IllegalArgumentException: Invalid parameter